manifests/init.pp

```diff
@@ -123,7 +123,9 @@
   validate_re($service_ensure, '^running|true|stopped|false$')
 
   anchor { 'sssd::begin': } ->
-  class { '::sssd::install': } ->
+  class { '::sssd::install':
+    ensure => $sssd_package_ensure,
```

Comment:
my local tests were failing because the var wasn't being read from the install class.  Is it a scoping issue with my version of Puppet (4.5)?  Agree that it's cleaner that way.  Maybe i can work around it in another way